canvas_modules/common-canvas/src/common-properties/components/field-picker/field-picker.scss

```diff
@@ -92,33 +92,16 @@
 	}
 }
 
-// This is for field-picker in properties-modal instead of flyout
-.properties-modal {
-	.properties-fp-top-row {
-		width: 100%;
-		margin-bottom: $spacing-03;
-		display: flex;
-		justify-content: space-between;
-
-		.properties-fp-button-label {
-			position: relative;
-			top: $spacing-03;
+// This is for field-picker in properties-modal, containerType="Modal"
+.properties-modal, .properties-tearsheet-panel.bx--modal {
+	.properties-fp-table {
+		.properties-fp-top-row {
+			display: flex;
+			justify-content: end;
```

Comment:
`npm run build` shows this warning in console -
<img width="846" alt="Screen Shot 2022-11-14 at 4 27 15 PM" src="https://user-images.githubusercontent.com/25124000/201796313-6e426629-54e1-401f-8629-177c81f0dad6.png">